src/pages/calculator/settings/index.tsx

```diff
@@ -1,8 +1,48 @@
+import React, { useCallback } from "react"
+
 import CalculatorLayout from "@/components/layouts/calculator"
 import { ComponentWithLayout } from "@/components/layouts/types"
+import { useAppDispatch, useAppSelector } from "@/store/hooks"
+import {
+  selectTravelerGender,
+  setTravelerGender,
+  TravelerGender,
+} from "@/store/settings/settingsSlice"
+import { Switch } from "@headlessui/react"
 
 export const SettingsPage: React.FC & ComponentWithLayout = () => {
-  return <div className="flex justify-center w-full max-w-5xl">Settings</div>
+  const dispatch = useAppDispatch()
+  // const [twin, setTwin] = useState<TravelerGender>(useAppSelector(selectTravelerGender))
```

Comment:
Let's get rid of comments that aren't necessary, fine for debugging and testing but lets remove for goal of merging into master